cross-platform/react-app/src/frontend/Screens/Hub/ProjectPicker.tsx

```diff
@@ -4,42 +4,70 @@
 *--------------------------------------------------------------------------------------------*/
 import React from "react";
 import { HorizontalPicker, useIsMountedRef } from "@itwin/mobile-ui-react";
-import { Project, ProjectsAccessClient, ProjectsQueryArg, ProjectsQueryFunction, ProjectsSearchableProperty, ProjectsSource } from "@itwin/projects-client";
+import { ITwin, ITwinsAccessClient, ITwinsAPIResponse, ITwinsQueryArg, ITwinSubClass } from "@itwin/itwins-client";
 import { IModelApp } from "@itwin/core-frontend";
 import { LoadingSpinner } from "@itwin/core-react";
 import { ButtonProps, HubScreenButton, HubScreenButtonList, HubScreenButtonListProps, i18n, presentError, PromiseUtil, SearchControl, useLocalizedString } from "../../Exports";
 
+type WithRequired<T, K extends keyof T> = T & { [P in K]-?: T[P] };
+export type Project = WithRequired<ITwin, "id">;
+
+enum ProjectsSource {
+  All = "",
+  Favorites = "favorites",
+  Recents = "recents"
+}
+
+interface ProjectsQueryResponse {
+  projects: Array<Project>;
+  next: ProjectsQueryFunction | undefined;
+}
+type ProjectsQueryFunction = () => Promise<ProjectsQueryResponse>;
+
 /**
- * Get a list of projects from the Bentley server.
+ * Get a list of iTwin Projects from the Bentley server.
  * @param source The source of the list: All, Favorites, or Recents
- * @param searchString A search string to search for if source is All.
- * @returns An object whose `projects` property is the list of projects, and whose `next` property
- * is an arrow function to fetch the next batch of projects if there are more than 100 results.
+ * @param search A search string
+ * @param skip Number of results to skip
+ * @returns A {@link ProjectsQueryResponse} object
  */
-async function getProjects(source = ProjectsSource.All, searchString = "") {
-  const client = new ProjectsAccessClient();
+async function getProjects(source: ProjectsSource, search = "", skip = 0): Promise<ProjectsQueryResponse> {
+  const client = new ITwinsAccessClient();
```

Comment:
You need to pass a possibly custom baseURL (with our API prefix).

Reply:
Something like:

```ts
  const baseUrl = `https://${process.env.ITMAPPLICATION_API_PREFIX ?? ""}api.bentley.com/itwins`;
```

Reply:
Is it necessary? The app already sets IMJS_URL_PREFIX which is used by itwins-client.

Reply:
Ah, I had forgotten about that. I guess it's not necessary.